catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -0,0 +1,259 @@
+import { useRouter } from 'next/router';
+import { FunctionComponent, ReactElement, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { pageDescriptions } from '@weco/common/data/microcopy';
+
+const NavBar = styled.nav`
+  border-bottom: 1px solid ${props => props.theme.color('warmNeutral.400')};
+  [aria-current='page'] {
+    &:after {
+      width: 100%;
+    }
+  }
+`;
+const NavList = styled.ul`
+  display: flex;
+  list-style: none;
+  list-style-position: inside;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+`;
+const NavItem = styled.li`
+  ${respondTo(
+    'headerMedium',
+    `
+    border-bottom: 0;
+    margin-right: calc(3vw - 20px);
+  `
+  )}
+
+  ${respondBetween(
+    'headerMedium',
+    'large',
+    `
+    font-size: 1.5vw;
+  `
+  )}
+
+  ${respondTo(
+    'xlarge',
+    `
+    margin-right: 1.4rem;
+  `
+  )}
+`;
+const NavLink = styled.a`
+  padding: 1.4rem 0.3rem;
+  display: inline-block;
+  text-decoration: none;
+  position: relative;
+  z-index: 1;
+  transition: color 400ms ease;
+  white-space: nowrap;
+
+  ${respondTo(
+    'headerMedium',
+    `
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  `
+  )}
+
+  &:after {
+    content: '';
+    position: absolute;
+    bottom: 0.1rem;
+    height: 0.2rem;
+    left: 0;
+    width: 0;
+    background: ${props => props.theme.color('yellow')};
+    z-index: -1;
+    transition: width 200ms ease;
+
+    ${respondTo(
+      'headerMedium',
+      `
+      // bottom: 0.9rem;
+    `
+    )}
+  }
+
+  &:hover,
+  &:focus {
+    &:after {
+      width: 100%;
+
+      // Prevent iOS double-tap link issue
+      // https://css-tricks.com/annoying-mobile-double-tap-link-issue/
+      @media (pointer: coarse) {
+        width: 0;
+      }
+    }
+  }
+`;
+
+const SearchLayout: FunctionComponent = ({ children }) => {
+  const router = useRouter();
+  const currentSearchCategory =
+    router.pathname === '/search'
+      ? 'overview'
+      : router.pathname.slice(router.pathname.lastIndexOf('/') + 1);
+
+  const basePageMetadata = {
+    openGraphType: 'website',
+    siteSection: 'collections',
+    jsonLd: { '@type': 'WebPage' },
+    hideNewsletterPromo: true,
+  } as const;
+
+  const defaultPageLayoutMetadata = {
+    ...basePageMetadata,
+    title: 'Search Page',
+    description: 'TBC',
+    url: { pathname: '/search', query: {} },
+  };
+
+  const [pageLayoutMetadata, setPageLayoutMetadata] = useState(
+    defaultPageLayoutMetadata
+  );
+  useEffect(() => {
+    const { query } = router.query;
+    switch (currentSearchCategory) {
+      case 'overview':
+        setPageLayoutMetadata(defaultPageLayoutMetadata);
+        break;
+      case 'exhibitions':
+        setPageLayoutMetadata({
+          ...basePageMetadata,
+          description: 'copy pending',
+          title: `${query ? `${query} | ` : ''}Exhibition Search`,
+          url: { pathname: '/search/exhibitions', query: router.query },
```

Comment:
a few of these are getting repeated, could they be part of `setPageLayoutMetadata` ? Again, could totally be part of a different metadata ticket.